pyracf/common/security_admin.py

```diff
@@ -54,6 +54,10 @@ class SecurityAdmin:
 
     _valid_segment_traits = {}
     _extracted_key_value_pair_segment_traits_map = {}
+    _racf_key_map = {}
```

Comment:
Add brief comment indicating that these structures are used for redaction?

Reply:
Sure.